enrollment-server/src/main/java/com/wultra/app/enrollmentserver/database/entity/OperationTemplate.java

```diff
@@ -18,17 +18,25 @@
 
 package com.wultra.app.enrollmentserver.database.entity;
 
-import lombok.Data;
+import lombok.*;
+import org.hibernate.Hibernate;
 
-import javax.persistence.*;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
 import java.io.Serializable;
+import java.util.Objects;
 
 /**
  * Entity representing an operation template.
  *
  * @author Petr Dvorak, petr@wultra.com
  */
-@Data
+@Getter
+@Setter
+@ToString
```

Comment:
I suggest to add a `@NoArgsConstructor` because the class implements serializable interface, which indicates serialization is possible. Some frameworks (e.g. Jackson ObjectMaper) fail to deserialize classes with missing no-args constructor.

Reply:
Good suggestion - added.